src/main/java/seedu/cc/logic/commands/entrylevel/FindCommand.java

```diff
@@ -68,20 +77,34 @@ public CommandResult execute(Model model, ActiveAccount activeAccount) {
             activeAccount.updateFilteredRevenueList(revenuePredicate);
         }
 
-        boolean isNoExpenseFoundWithOnlyExpensePredicate =
-            activeAccount.getFilteredExpenseList().size() == 0 && expensePredicate != null && revenuePredicate == null;
-        boolean isNoRevenueFoundWithOnlyRevenuePredicate =
-            activeAccount.getFilteredRevenueList().size() == 0 && revenuePredicate != null && expensePredicate == null;
-        boolean isNoEntryFound = activeAccount.getFilteredExpenseList().size() == 0
-                            && activeAccount.getFilteredRevenueList().size() == 0;
+        List<Expense> expenseList = activeAccount.getFilteredExpenseList();
+        List<Revenue> revenueList = activeAccount.getFilteredRevenueList();
+
+        boolean noExpenseFound =
+            noExpenseFoundWithOnlyExpensePredicate(expenseList);
+        boolean noRevenueFound =
+            noRevenueFoundWithOnlyRevenuePredicate(revenueList);
+        boolean noEntryFound = noEntryInListFound(expenseList) && noEntryInListFound(revenueList);
 
-        if (isNoEntryFound || isNoExpenseFoundWithOnlyExpensePredicate || isNoRevenueFoundWithOnlyRevenuePredicate) {
+        if (noEntryFound || noExpenseFound || noRevenueFound) {
```

Comment:
I think these names are not boolean names!. Please rename to boolean (i.e. is No ExpenseFound, etc)